src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -93,6 +93,7 @@
 	display: block;
 }
 
+.monaco-workbench .pane-body.integrated-terminal .terminal-group .monaco-split-view2.horizontal .split-view-view:first-child .xterm,
```

Comment:
not sure why, but this at the moment only applies to the last created terminal (as opposed to the leftmost)